mmv1/templates/terraform/examples/firebase_app_hosting_backend_basic.tf.tmpl

```diff
@@ -0,0 +1,76 @@
+# Service account setup only required once per project.
+resource "google_service_account" "service_account" {
+  provider = google-beta
+
+  project = "{{index $.TestEnvVars "project_id"}}"
+
+  # Must be firebase-app-hosting-compute
+  account_id                   = "firebase-app-hosting-compute"
+  display_name                 = "Firebase App Hosting compute service account"
+
+  # Do not throw if already exists
+  create_ignore_already_exists = true
```

Comment:
What is the purpose of including this? Tests should be deleting this account when they complete

Reply:
See comment above (we have to use this name so sweepers can't reach it).